workflow/state-machines.rst

```diff
@@ -32,7 +32,7 @@ Below is the configuration for the pull request state machine.
         framework:
             workflows:
                 pull_request:
-                    type: 'state_machine'
+                    type: 'single_state'
```

Comment:
I see my problem -- I didn't realize that workflows now had a type 'state_machine', and that if you wanted a state machine you had to set the marking_store type to single_state.